app/components/progressive_image/progressive_image.test.js

```diff
@@ -22,22 +22,34 @@ describe('ProgressiveImage', () => {
         tintDefaultSource: false,
     };
 
-    test('should match snapshot when just a image loads', () => {
-        const wrapper = shallow(
-            <ProgressiveImage {...baseProps}/>
-        );
+    test('should match snapshot', () => {
+        const wrapper = shallowWithIntl(<ProgressiveImage {...baseProps}/>);
         expect(wrapper.getElement()).toMatchSnapshot();
     });
 
-    test('should match snapshot when no thumbnail', () => {
-        const props = {
-            ...baseProps,
-            thumbnailUri: null,
-        };
+    test('should load image', () => {
+        const wrapper = shallowWithIntl(<ProgressiveImage {...baseProps}/>);
+        const instance = wrapper.instance();
+        const load = jest.spyOn(instance, 'load');
+        load();
+        expect(load).toHaveBeenCalled();
```

Comment:
you are calling load explicitly in the above line, so it seems obvious is going to get called. @saturninoabril can you provide assistance in how he can test that `load` is being called in `componentDidMount`?